README.md

```diff
@@ -1,47 +1,40 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# pv-sites-mobile
 
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+## About
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
+With site-level forecasts—nowcasting for individual solar locations—OCF is in the process of bringing the nowcasting experience to the broader public, to give people more power over their own solar setup. Smart home operators, solar PV asset operators, offtakers, and individual home owners can all make use of better forecasting, and in turn, OCF’s impact reaches more people and helps further optimize the contribution of solar power to the electricity grid.
```

Comment:
think we should reorder the list of stakeholders based on our P's in PRD

Reply:
also maybe "we" would be better here than "OCF" since this repo is under OCF anyways—but I really don't know maybe just try it out and see which one sounds better